docs/languages/julia.md

```diff
@@ -0,0 +1,71 @@
+# Julia in Visual Studio Code
+
+The Visual Studio Code extension for Julia includes built-in dynamic autocompletion, inline results, plot pane, integrated REPL, variable view, code navigation, and many other advanced language features.
+
+![Julia VS Code Overview](images/julia/overview.png)
+
+Most of these features just work out of the box, while some may require basic configuration to get the best experience. This page summarizes the Julia features that the Julia VS Code extension ships with. For a more in-depth guide on how these features work and can be configured, see [The Julia VS Code Docs](https://www.julia-vscode.org/docs/stable/).
```

Comment:
see the [Julia VS Code documentation]().
Looking at julia-vscode.org, I see that sometimes the documentation uses "VSCode", with no space.
Our preference is to say "Visual Studio Code" on first reference in a topic and then "VS Code" with a space after that.

Reply:
Just fixed that across the docs site, will do the other page as well. Thanks for the reminder

Reply:
We should have a new version of the docs tagged soon to propagate these changes. Please let me know if you see other places where the naming is off!